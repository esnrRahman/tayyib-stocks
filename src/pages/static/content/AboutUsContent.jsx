import React, { Fragment } from 'react';

const AboutUsContent = () => (
  <Fragment>
    <h1>
      Our Story
    </h1>
    <p>
      We are a group of Computer and Software Engineers who have an active interest in the
      stock market. Before going ahead and investing in stocks, however, we asked ourselves
      &#34;Is buying shares in any public company, regardless of the nature of their business,
      halal in Islam?&#34;.
      <br />
      <br />
      Common sense dictates that there must be certain Islamic moral conditions a company has
      to meet for it to be acceptable to be a part owner of it.
      <br />
      <br />
      We began to research on this issue and learnt that, as expected, there are certain
      criteria a company has to fulfill for it to be Islamically permissible to invest in. But
      finding out which particular companies satisfy these criteria was not as simple as we had
      hoped it would be.
      <br />
      <br />
      To solve this problem, we put together this small web application for everyone in the Muslim
      community who has an interest in the stock market and cares about ensuring their financial
      dealings are halal and tayyib.
      <br />
      <br />
      May Allah accept from us this small effort and forgive us for our mistakes and shortcomings.
    </p>
    <h2>
      Vision
    </h2>
    <p>
      A World Where it is Very Simple for Every Muslim Investor to Make an Informed, Halal and
      Tayyib Investment in the Stock Market.
    </p>
    <h2>Mission</h2>
    <p>
      To make finding out whether a stock is Halal and Tayyib to invest in a
      &#34;one-click&#34; process.
    </p>
    <br />
  </Fragment>
);

export default AboutUsContent;
