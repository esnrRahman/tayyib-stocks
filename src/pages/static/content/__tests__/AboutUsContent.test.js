import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import AboutUsContent from '../AboutUsContent';

describe('<AboutUsContent />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AboutUsContent />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
