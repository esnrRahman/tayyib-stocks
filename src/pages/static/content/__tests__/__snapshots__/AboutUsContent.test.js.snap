// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutUsContent /> renders correctly 1`] = `
<Fragment>
  <h1>
    Our Story
  </h1>
  <p>
    We are a group of Computer and Software Engineers who have an active interest in the stock market. Before going ahead and investing in stocks, however, we asked ourselves "Is buying shares in any public company, regardless of the nature of their business, halal in Islam?".
    <br />
    <br />
    Common sense dictates that there must be certain Islamic moral conditions a company has to meet for it to be acceptable to be a part owner of it.
    <br />
    <br />
    We began to research on this issue and learnt that, as expected, there are certain criteria a company has to fulfill for it to be Islamically permissible to invest in. But finding out which particular companies satisfy these criteria was not as simple as we had hoped it would be.
    <br />
    <br />
    To solve this problem, we put together this small web application for everyone in the Muslim community who has an interest in the stock market and cares about ensuring their financial dealings are halal and tayyib.
    <br />
    <br />
    May Allah accept from us this small effort and forgive us for our mistakes and shortcomings.
  </p>
  <h2>
    Vision
  </h2>
  <p>
    A World Where it is Very Simple for Every Muslim Investor to Make an Informed, Halal and Tayyib Investment in the Stock Market.
  </p>
  <h2>
    Mission
  </h2>
  <p>
    To make finding out whether a stock is Halal and Tayyib to invest in a "one-click" process.
  </p>
  <br />
</Fragment>
`;
